virttest/qemu_devices/qdevice_format.py

```diff
@@ -89,13 +89,6 @@ def __init__(self):
                     "physical_block_size": self._str_to_dec,
                     "logical_block_size": self._str_to_dec,
                 },
-                "usb_driver": {
-                    "port": self._str,
-                    "serial": self._on,
-                    # min_io_size, opt_io_size from device ("driver": "usb-storage")
-                    "min_io_size": self._str_to_dec,
-                    "opt_io_size": self._str_to_dec,
```

Comment:
hi @nickzhq , I am curious about how to handle these two options, the type is <size>, e.g.
min_io_size=<size>     -  (default: 0)
opt_io_size=<size>     -  (default: 0)
but I didn't find the mapping in self._type_func_mapping

Reply:
> hi @nickzhq , I am curious about how to handle these two options, the type is <size>, e.g. min_io_size= - (default: 0) opt_io_size= - (default: 0) but I didn't find the mapping in self._type_func_mapping

Thanks for response. After I discussed with @YongxueHong , we decided to dynamically get total usb device types from qemu since there are too many usb devices. If some args are incorrect from qemu, feature owners  are able to code at here in a new patch. In other words, they can manage this mapping by themselves.

Thanks!

Reply:
Thanks for the quick response, I learnt from the code that we'll get the options from the output of qemu-kvm --device xxx,?, my concern is we used to support the conversions for both min_io_size and opt_io_size for the usb-storage driver, with the patch, the support was gone, so I am afraid that it could introduce regressions when running the usb-storage related test cases(Not sure if we have these cases, but from the original code, it looks like we have as you made specific comments for usb-storage driver)